### ISSUE TITLE
Apocalyptica

### ISSUE THREAD

**Issue by NathanFambrough** [siggame/MegaMinerAI-Game-Ideas#52]
# HEADER
In Which Nuclear Power Plants Complicate Things

## Story
Once upon a time, The Apocalypse happened, killing almost- What do you mean 'What kind'? _The_ Apocalypse. Don't ask questions.

Most of the people on Earth died, governments collapsed, and everyone gathered together in warromg tribes. Society is rebuilding, but it doesn't take long for everyone to realize the threat posed to everyone. The nuclear power plants.

Each of these require maintenance to ensure they don't go critical, lest they make a beautiful mushroom cloud, and irradiate the environment with nuclear fallout.

## TL;DR
You hate the other guys, they hate you, but you all hate being irradiated even more.

## Win Conditions
The goal is to destroy the enemy's tribe. In the event this has not occurred after (ARBITRARY TURNCOUNT), the winner shall be decided by the following conditions:

- Who has the most captured power plants
- Who has the most people
- Coin toss

## Gameplay
Haven't put much thought into this yet, unfortunately.
This is still a very much rough draft.

**Comment by JeffreyStrahm** [comment]
Try and say nuclear power plants, when I read it at first I thought it was about giant angry irradiated plants. Otherwise a solid outline, there would be a bit of building. It's an interesting mechanic having an outside element that you have to manage to stay alive. Balancing their effects if you fail to do it would be interesting. As well as how to get more people and the types of fighters.

What tech level are we going for here? Guns or bows and arrows?

**Comment by NathanFambrough** [comment]
It's up for debate. Whichever is more convenient to program, but I'm thinking guns.

Depending on the scale of things, a nuclear power plant going critical can spell the end of the match, or just the irradiation of that region.

I was thinking of radiation causing a sort of poison effect, where everyone gradually loses health every turn they're inside. Until a reactor goes critical, the only place one _might_ encounter radiation would be inside the plant. Certain unit types could potentially be immune to radiation, like a HAZMAT unit or a Technician or something.

If we're treating the power plant as a game-ender, then the 'benefit' of keeping it from going critical is the continuation of the match. If we're treating it as a map feature, then some benefits might be... Increased energy for the victor, and being able to post riflemen on top? Runs the risk of snowballing, though.
